cmd/cluster/install.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright © 2023 NAME HERE <EMAIL ADDRESS>
+*/
+package cluster
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"openGemini-UP/pkg/install"
+)
+
+// installCmd represents the install command
+var installCmd = &cobra.Command{
+	Use:   "install",
+	Short: "install cluster",
+	Long:  `Install an openGemini cluster based on configuration files and version numbers.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		ops, err := getClusterOptions(cmd)
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+
+		err = InstallCluster(ops)
+		if err != nil {
+			fmt.Println(err)
```

Comment:
Is return required? Like https://github.com/openGemini/openGemini-UP/pull/11/files#diff-d1bbf7dccf2ad778d6da48b735e0db27a5b138c80327fe6398ceb3fc2c65eff0R22

Reply:
If the `getClusterOptions` had `error`, we shoule not continue to execute `InstallCluster`.